test/built-ins/SharedArrayBuffer/prototype/growable/this-is-arraybuffer.js

```diff
@@ -18,7 +18,7 @@ var growable = Object.getOwnPropertyDescriptor(
 );
 
 var getter = growable.get;
-var ab = new SharedArrayBuffer(4);
+var ab = new ArrayBuffer(4);
```

Comment:
The error messages below are still wrong.

Reply:
I'm moving a bit too fast for my own good. That's fixed now, too.